geonode/notifications_helper.py

```diff
@@ -129,7 +129,7 @@ def get_notification_recipients(notice_type_label, exclude_user=None, resource=N
                         not user.has_perm('view_resourcebase', resource.get_self_resource()):
                     exclude_users_ids.append(user.id)
                 if user.pk == resource.owner.pk and \
-                        not notice_type_label.endswith("_comment"):
+                        not (notice_type_label.endswith("_comment") or notice_type_label.endswith("_updated")):
```

Comment:
Strange, that case should be already covered by `exclude_users_ids.append(exclude_user.id)`. Did you experienced such misbehavior?

Reply:
@afabiani Good point. It currently is not used in e.g. layer creation and will be `none` but I think it would be better to use that to exclude the signal firing user (= the one uploading, updateding, commenting).